packages/web/components/assets/generate-gradient.ts

```diff
@@ -0,0 +1,13 @@
+export default function generateGradient(num?: number) {
```

Comment:
It would be better to pull this file from `components` and put other folder like `utils`.

Reply:
Moved it to `components/utils.ts`